app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt

```diff
@@ -250,11 +270,15 @@ class EditKeyboardFragment : BaseFragment() {
         binding?.editConfirmation?.dialogNegativeButton?.let {
             it.text = getString(R.string.discard)
             it.action = {
-                parentFragmentManager
-                    .beginTransaction()
-                    .replace(R.id.settings_fragment_container, EditPresetsFragment())
-                    .addToBackStack(null)
-                    .commit()
+                if (isCategory) {
+                    parentFragmentManager.popBackStack()
+                } else {
+                    parentFragmentManager
```

Comment:
Should this one also be `parentFragmentManager.popBackStack()`?

Reply:
yes i think so, thank you!